backend/src/main/java/gov/cdc/usds/simplereport/api/converter/FhirConverter.java

```diff
@@ -940,7 +957,10 @@ public Bundle createFhirBundle(
             .device(convertToDevice(testEvent.getDeviceType()))
             .specimen(
                 convertToSpecimen(
-                    testEvent.getSpecimenType(), uuidGenerator.randomUUID(), null, null))
+                    testEvent.getSpecimenType(),
+                    uuidGenerator.randomUUID(),
+                    specimenCollectionDate,
+                    dateTested))
```

Comment:
does using `dateTested` here work?

I thought `Specimen.receivedTime` was getting mapped to SPM-18, which I think is the one that uses `specimen_collection_date_time - 6 seconds` in the Covid pipeline.
or are we not worrying about this one matching?

Reply:
thanks for making the changes